File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/HeaderedPanel/Features/HeaderedPanelIconPropertyPage.vb
Namespace CompanionKit.Controls.HeaderedPanel.Features

	Public Class HeaderedPanelIconPropertyPage

        Public Sub New()
            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()
        End Sub
        ''' <summary>
        ''' Handles CheckedChanged event for RadioButton controls
        ''' </summary>
        Private Sub RadioButton_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles mobjRadioButton3.CheckedChanged, mobjRadioButton2.CheckedChanged, mobjRadioButton1.CheckedChanged
            ' If RadioButton is checked
            If DirectCast(sender, Gizmox.WebGUI.Forms.RadioButton).Checked Then
                ' Set icon for HeaderedPanel using RadioButton.Image property
                Me.mobjHeaderedPanel.Icon = DirectCast(sender, Gizmox.WebGUI.Forms.RadioButton).Image
            End If
        End Sub

	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/HeaderedPanel/Features/HeaderedPanelIconPropertyPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.HeaderedPanel.Features
{
    public partial class HeaderedPanelIconPropertyPage : UserControl
    {
        public HeaderedPanelIconPropertyPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles CheckedChanged event for RadioButton controls
        /// </summary>
        private void RadioButton_CheckedChanged(object sender, EventArgs e)
        {
            // If RadioButton is checked
            if (((Gizmox.WebGUI.Forms.RadioButton)sender).Checked)
            {
                // Set icon for HeaderedPanel using RadioButton.Image property
                mobjHeaderedPanel.Icon = ((Gizmox.WebGUI.Forms.RadioButton)sender).Image;
            }
        }
    }
}